### ISSUE TITLE
custom error middleware always execute after default

### ISSUE THREAD

**Issue by Diluka** [pleerock/routing-controllers#23]
I have custom error middleware to handle errors and decide different way to response. But neither Infinity nor -Infinity I set to the `priority` option, the default error middleware send the response with status 500 before my error middleware. My error middleware seems to be useless.

how can I solve this?

**Comment by Diluka** [comment]
Seems it's the same pleerock/routing-controllers#17

**Comment by Diluka** [comment]
```
ExpressDriver.prototype.handleError = function (error, action, options) {
    options.next(error);
};
```
@peterjp80
This is the temporary way to disable the default error handler inside. This is invoked immediately atfer action raise an error and send the respone, so the next error middlewares can do nothing but just logging.

**Comment by 19majkel94** [comment]
@Diluka is that still not working for you? Can this issue be closed, the PR #36 has solved your problem?
